src/web100srv.c

```diff
@@ -820,11 +626,7 @@ int run_test(tcp_stat_agent *agent, Connection *ctl, TestOptions *testopt,
   double c2sspd;          // average throughput as calculated by C->S test
   double s2c2spd;         // average throughput as calculated by midbox test
   double realthruput;     // total send throughput in S->C
-  double acksratio;       // ratio of acks over packets sent
   double aspd = 0;
```

Comment:
True, but it is used by old code which prints things out.  The stuff I deleted was commented out in the main trunk, and then I accidentally uncommented it.